zilliqa/src/p2p_node.rs

```diff
@@ -242,6 +235,7 @@ impl P2pNode {
         if let Some((peer, address)) = &self.config.bootstrap_address {
             self.swarm.dial(
                 DialOpts::peer_id(*peer)
+                    .override_role() // hole-punch
```

Comment:
Does this mean we have NAT punch-through? Or does this comment mean something else?

Reply:
This turns on the NAT punch-thru capability if needed. I'm unsure if it's required, but I don't think there's any harm.